src/components/Login/LoginOTPCard.tsx

```diff
@@ -88,11 +88,6 @@ export const LoginOTPCard: FunctionComponent<LoginOTPCard> = ({
           setAuthInfo(response.sessionToken, response.ttl.getTime(), endpoint);
           setFeatures(versionResponse.features);
           setProducts(versionResponse.policies);
-        case "MERCHANT":
```

Comment:
Hmm thinking if we want to just check for the existence of FLOW_TYPE rather than switch thru the different envs here, since actually at this point this component doesnt care, it just wants to know whether to save the info or not.

Reply:
agree with this, since envs are check in login container, any possibility that the FLOW_TYPE is some invalid string?

Reply:
Technically it will be invalid if we change the types of strings accepted (e.g. COLLECTION vs MERCHANT). But then again, this switch just checks whether to save info or not. So regardless, once FLOW_TYPE is defined, I think it's safe to say that the info should be saved